astropy/units/function/core.py

```diff
@@ -6,6 +6,7 @@
 
 import numpy as np
 
+import astropy.units as u
```

Comment:
this messes up things, so rather import the specific classes you use in the line below.